gradle.properties

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-release_version=3.29.2
+release_version=3.29.3
```

Comment:
Please update the minor version according API has been extended

Reply:
done